.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -28,6 +28,9 @@
       - [URL to each sample source, if applicable]
     - Sample license(s):
   - [ ] I have included a syntax highlighting grammar: [URL to grammar repo]
+      <!-- Setting a color is only needed for 'programming' and 'markup' languages and will default to `#cccccc` if not specified. -->
```

Comment:
This comment contradicts itself: either a coloUr is "needed" by `programming`/`markup`-type languages, or it defaults to `#cccccc`.

Also, this explanation doesn't take into account languages that users have opted to include in their language-stats (with `language-detectable`). Since having a colour benefits `prose`- and `data`-type languages this way, I'd keep this on the short-and-sweet side:

```suggestion
      <!-- Setting a colour is strongly recommended, but optional: `#cccccc` is used by default -->
```

Reply:
Makes sense to me, but sadly, it's got to be 🇺🇸 "color" cos that's the GitHub standard.